repspark_at/test_cases/login.py

```diff
@@ -6,11 +6,19 @@
 
 class AppLogin(SetUpClass):
     def test_login(self):
-        LoginPage.set_user_name(self, "qwerty@mail.com")
-        LoginPage.set_password(self, "root")
-        LoginPage.click_login_button(self)
+        login_page = LoginPage(self.driver)
+        login_page.set_user_name("qwerty@gmail.com")
+        login_page.set_password("root")
+        login_page.click_login_button()
+
+        # LoginPage.set_user_name(self, "qwerty@mail.com")
+        # LoginPage.set_password(self, "root")
+        # LoginPage.click_login_button(self)
 
         BrowsePage.check_title(self)
```

Comment:
@qaivanpleten you have to use objects here too. Look for next line. It has the correct implementation.

Reply:
@extsoft 
done